static/js/activity.js

```diff
@@ -257,7 +257,7 @@ exports.update_users = function (user_list) {
         $.each(user_info, function (index, user) {
             var user_index = all_users.indexOf(user.email);
             $('#user_presences').find('[data-email="' + user.email + '"]').remove();
-            $('#user_presences li').eq(user_index + 1).before(templates.render('user_presence_row', user));
+            $('#user_presences li').eq(user_index).before(templates.render('user_presence_row', user));
```

Comment:
Well, I happen to have tested this, so I'm pretty sure the new logic is correct.  But here's the explanation. Suppose we have:
* The old list is say [a, b, c, d, e, f]
* f is moving up and we want to get to the new list [a, b, f, c, d, e]

Then user_index will be 2, and we remove f, making the current list [a, b, c, d, e].  Then we want to insert f just before the "c", which in the current list is at index 2 (though it will be at index 2+1 after we put f in).  So we want to insert before index user_index, not `user_index + 1`.

Does that make sense?